ctam/interfaces/fw_update_ifc.py

```diff
@@ -165,18 +165,21 @@ def ctam_stage_fw(
         if partial == 0 and pushtargets:
             self.ctam_pushtargets()
         JSONFWFilePayload = self.get_JSONFWFilePayload_file(image_type=image_type, corrupted_component_id=corrupted_component_id)
-        if not os.path.isfile(JSONFWFilePayload):
+        if not JSONFWFilePayload or not os.path.isfile(JSONFWFilePayload):
             self.test_run().add_log(LogSeverity.DEBUG, f"Package file not found at path {JSONFWFilePayload}!!!")
             return False, "", ""
         if self.dut().is_debug_mode():
             print(JSONFWFilePayload)
-        uri = self.dut().uri_builder.format_uri(
-            redfish_str="{BaseURI}/UpdateService", component_type="GPU"
-        )
+        status, uri, is_multipart = self.get_update_uri()
```

Comment:
We are not getting the proper uri with GPUMC, so test is failing for all FW Update test cases. Need to format GPUMC as prefix to uri using redfish uri builder.

Reply:
we shall approve this PR for now. And we shall create a separate PR with the new changes.